README.md

```diff
@@ -10,37 +15,35 @@ pip install ismatec
 Usage
 =====
 
-### Command Line
+## Command Line
 
 ```
-$ ismatec <serial-to-ethernet-ip> --port <port> --channel 1
 $ ismatec /dev/ttyUSB0 --channel 2
+$ ismatec tcp://192.168.1.100:23 --channel 1
 ```
 
-This will print the current flowrate for the selected channel, using either an
-Ethernet-to-serial adapater, or a serial port.
+This will print the current flow rate for the selected channel, using either the serial port or an ethernet-to-serial adapter.
 
-### Python
+## Python
 
-This uses Python ≥3.5's async/await syntax to asynchronously communicate with
-a RegloICC pump. For example:
+This uses Python ≥3.5's async/await syntax to asynchronously communicate with a Ismatec® Reglo ICC pump. For example:
 
 ```python
 import asyncio
 from ismatec import Pump
 
 async def get():
-    async with Pump(('serial-to-ethernet-ip', 23)) as pump:
-        print(await pump.get_pump_version())
```

Comment:
I thought the "if tuple then IP" API was clunky and prone to frustrating users.

Reply:
Yeah, it's from the original author.  I'm generally happy with your rewrite here, but (1) want to test on the real pump and (2) want to maintain some consistency with our other drivers.